mypy/test/data.py

```diff
@@ -158,6 +158,7 @@ def __init__(self,
                  output_files: List[Tuple[str, str]],
                  expected_stale_modules: Optional[Set[str]],
                  expected_rechecked_modules: Optional[Set[str]],
+                 native_sep: bool = False,
```

Comment:
Could you explain what this is used for? I don't see it used beyond creating an instance variable.